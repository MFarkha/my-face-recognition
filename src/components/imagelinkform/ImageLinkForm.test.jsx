import React from 'react';
import {render, screen} from '@testing-library/react';

import ImageLinkForm from './ImageLinkForm';

it('renders and displays image link form correctly', () => {
    const { container } = render (<ImageLinkForm />);
    expect(container).toMatchSnapshot();
    expect(screen.getByRole('textbox')).toBeInTheDocument();
    expect(screen.getByRole('button', { value: /detect/i })).toBeInTheDocument();
})