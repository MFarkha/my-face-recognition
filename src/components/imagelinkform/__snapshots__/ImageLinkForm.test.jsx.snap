// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders and displays image link form correctly 1`] = `
<div>
  <div>
    <p
      class="f3"
    >
      This Magic Brain will detect faces in your pictures. Give it a try!
    </p>
    <div
      class="center"
    >
      <div
        class="center form pa4 br3 shadow-5"
      >
        <input
          class="f4 pa2 w-70 center"
          id="inputUrl"
          type="text"
        />
        <button
          class="w-30 grow f4 link ph3 pv2 dib white bg-light-purple"
        >
           Detect 
        </button>
      </div>
    </div>
  </div>
</div>
`;
